tests/network/vmi_multus.go

```diff
@@ -332,26 +353,43 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		})
 
 		Context("VirtualMachineInstance with Linux bridge plugin interface", func() {
-			table.DescribeTable("should be able to ping between two vms", func(interfaces []v1.Interface, networks []v1.Network, ifaceName string) {
+			table.DescribeTable("should be able to ping between two vms", func(interfaces []v1.Interface, networks []v1.Network, ifaceName, cidr1, cidr2 string) {
+				if cidr2 == "dhcp" {
```

Comment:
What if it's `cidr1` that holds dhcp ? No need to provision the NAD ?

Reply:
oops. Thanks for the catch.
changed to 
`staticIPVm1 == "" | | staticIPVm2  == "`